### PR TITLE
Fix/registry logging

### PR DESCRIPTION
**Pull request by andre-merzky**
This makes ZMQ logging more sane to handle.

### CODE REVIEW

File: src/radical/utils/zmq/utils.py [hunk 1]
```diff
@@ -8,6 +8,10 @@
 from ..misc import ru_open
 
 
+# NOTE: this is ignoring `RADICAL_LOG_LVL` on purpose
+LOG_ENABLED = os.environ.get('RADICAL_ZMQ_LOG', '0').lower() in ['1', 'true']
```
Comment (mtitov) [inline src/radical/utils/zmq/utils.py:12]: I got confused with the necessity of introducing this.. I see that you've updated `ns` for loggers (some of them were already set correctly), which makes it easy and right way to control all of this with `RADICAL_UTILS_ZMQ_LOG_LVL`.
As well:
- all debug messages here are level 9, thus they will never appear for the regular debugging and controlling their appearance with the env variable above is straightforward;
- with the proposed setup it is not possible to turn off zmq-related log messages completely;

Comment (mtitov) [reply src/radical/utils/zmq/utils.py:12]: one more thing, we can keep an explicit default value for `RADICAL_UTILS_ZMQ_LOG_LVL` in `DefaultConfig` (`utils_default.json`)

File: src/radical/utils/zmq/registry.py [hunk 2]
```diff
@@ -86,6 +97,8 @@ def put(self, key: str, val: Any) -> None:
         path  = elems[:-1]
         leaf  = elems[-1]
 
+        self._log.info('put %s: %s', str(key), str(val))
```
[the remark this answers is not in the document]
Comment (andre-merzky) [reply src/radical/utils/zmq/registry.py]: Thanks

File: src/radical/utils/zmq/registry.py [hunk 3]
```diff
@@ -117,6 +130,8 @@ def get(self, key: str) -> Optional[str]:
             this = dict()
 
         val = this.get(leaf)
+
+        self._log.info('get %s: %s', str(key), str(val))
```
[the remark this answers is not in the document]
Comment (andre-merzky) [reply src/radical/utils/zmq/registry.py]: indeed

File: src/radical/utils/zmq/registry.py [hunk 4]
```diff
@@ -136,13 +151,19 @@ def keys(self, pwd: Optional[str] = None) -> List[str]:
...
+        self._log.info('keys: %s', keys)
+
+        return keys
 
 
     # --------------------------------------------------------------------------
     #
     def delitem(self, key: str) -> None:
 
+        self._log.info('del: %s', key)
```
[the remark this answers is not in the document]
Comment (andre-merzky) [reply src/radical/utils/zmq/registry.py]: fixed

File: src/radical/utils/zmq/utils.py (hunk 1 above)
Comment (andre-merzky) [reply src/radical/utils/zmq/utils.py:12]: Not really a necessity, but convenience: the ns is not consistently used, specifically when some ZMQ subscriber etc uses a 3rd party logger (such as the agent logger etc).  Also, the `bulk_log` method is a bit expensive even when not logging anything, so I wanted to shield zmq logging separately.

**Review by mtitov — APPROVED** [review]
LGTM!

**Pull request merged**